fallbear-cmd

```diff
@@ -2,63 +2,85 @@
 
 echo_debug() {
     if [ -n "$PVDEBUG" ]; then
-	echo $@
+        echo "$@" >&2
     fi
 }
 
 lxc_pid() {
     target=$1
-    nsenter -m/proc/1/ns/mnt \
-	    lxc-info $target | grep PID: | sed -e 's/^[^0-9]*//'
+    nsenter --target 1 --mount -- lxc-info --no-humanize --pid "$target"
 }
 
 target=$USER
 p=1
 plat=${USER##*@}
 plat_tty=${USER%@$plat}
 
-
 if [ "$plat" = "$plat_tty" ]; then
-	plat_tty=
+    plat_tty=
 fi
 
-if ! [ "$plat" = "/" -o "$plat" = "_pv_"  ]; then
-    p=`lxc_pid $plat`
+case "$plat" in
+"/"|"_pv_")
```

Comment:
not an issue, but elsewhere in our code base we indent the case matches once and the code + ;; twice